src/language/c-helpers.rkt

```diff
@@ -0,0 +1,25 @@
+#lang typed/racket/no-check
+(require racket/string "../configuration.rkt")
+(provide (all-defined-out))
+
+(: define-line (Output-Port -> ((Syntaxof Symbol) Syntax -> Void)))
+(define ((define-line in) s v)
+  (define n? (syntax->datum v))
+  (define str (string-replace (symbol->string (syntax-e s)) #rx"-|/" "_"))
```

Comment:
This isn't a fully general solution for all symbols that would work.  Perhaps white listing is a better solution. You could use `#px"[^\\w]"` which matches any character that is not alpha numeric.

Reply:
Done.